### PR TITLE
Translation of webassembly chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of [#586](https://github.com/HTTPArchive/almanac.httparchive.org/issues/586)

### CODE REVIEW

File: src/content/ja/2022/webassembly.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+これは、クロールで発見したモジュールの数が控えめに減少したことを表しており、デスクトップで16％、モバイルで12％の減少となっています。これは必ずしもWebAssemblyが衰退していることを意味するものではなく、この変化を解釈する場合、以下の点に注意する必要があります。
+
+- WebAssemblyを使用してあらゆる種類のWebベースのコンテンツを作成できますが、その主な利点は、大規模なコードベースを持つより複雑な業務用アプリケーションで、何年も前のものであることが多いです（例：Google Earth、Photoshop、AutoCADなど）。これらのWeb「アプリ」は、Webサイトほど多くはなく、WebAssemblyがあまり普及していないホームページを主な対象とするAlmanacのクロールでは常に利用できるわけではありません。
+- 後のセクションで見るように、私たちが見るWebAssemblyの使用の多くは、比較的少数のサードパーティライブラリから来るものです。その結果、これらのライブラリのどれか1つでも小さな変更があれば、見つかるモジュールの数に大きな影響を与えることになります。
+
+モバイルブラウザに提供されたWebAssemblyモジュールがわずかに少ない（-13％）ことがわかりました。これは、一般的に優れたサポートを持つモバイルブラウザのWebAssembly能力を反映しているわけではありません。むしろ、[プログレッシブ・エンハンスメント](https://developer.mozilla.org/ja/docs/Glossary/Progressive_Enhancement)の標準的な実践によるものと思われます。この場合、WebAssemblyを必要とするより高度な機能は、モバイルユーザーにはサポートされていません。
+
+{{ figure_markup(
+  caption="Number of Wasm responses.",
```
Comment (tunetheweb) [inline src/content/ja/2022/webassembly.md:45]: Could you translate this caption?

File: src/content/ja/2022/webassembly.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  image="counts.png"
+  )
+}}
+
+WebAssemblyのモジュールをハッシュ化することで、デスクトップ用の3,204個のモジュールのうち、いくつがユニークなモジュールなのかを特定できます。モジュールの重複を排除することで、総数はおよそ10分の1になり、ユニークなモジュールはデスクトップブラウザに383個、モバイルに310個提供されています。これは、異なるウェブサイトが同じWebAssemblyコードを利用し、おそらく共有モジュールによって再利用されていることを意味します。
+
+wasmリクエストのかなりの割合がクロスオリジンであり、再利用されているという考えをさらに強めています。とくに、これは昨年から大幅に増加しています（67.2％対55.2％）。
+
+{{ figure_markup(
+  caption="Cross-origin WebAssembly usage.",
```
Comment (tunetheweb) [inline src/content/ja/2022/webassembly.md:59]: And this one?

**Review by tunetheweb — CHANGES_REQUESTED** [review]

File: src/content/ja/2022/webassembly.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2022/webassembly.md]: Thanks, I translated.

File: src/content/ja/2022/webassembly.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2022/webassembly.md]: Thanks, I translated.

**Review by ksakae1216 — COMMENTED** [review]
@tunetheweb Thanks, I translated.
Please check.

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**